src/Workspace/index.jsx

```diff
@@ -256,20 +256,50 @@ class Workspace extends React.Component {
       (element.type === 'ContainersReady' && element.status === 'False')
     ))) {
       workspaceLaunchStepsConfig.currentIndex = 2;
-      if (workspaceStatusData.containerStates.some((element) => (
+      workspaceLaunchStepsConfig.steps[2].description = 'In progress';
+      const cs = workspaceStatusData.containerStates;
+      console.log('container status length:' + cs.length);
+
+      if (cs.some((element) => (
         (element.state && element.state.terminated)
       ))) {
         workspaceLaunchStepsConfig.steps[2].description = 'Error';
         workspaceLaunchStepsConfig.currentStepsStatus = 'error';
-      } else {
-        workspaceLaunchStepsConfig.steps[2].description = 'In progress';
+      }
+      else
+      {
+        if(cs.length > 1)
```

Comment:
why not display these info if there is only 1 container?

Reply:
Changed the compare to > 0